data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,307 @@
+
+AUDIO
+===============
+
+MUSIC
+---------------
+
+cry from elensefar (pluft) - CC BY-SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver), remixed by dalas - CC BY-SA 4.0 (both)
+    https://forums.wesnoth.org/viewtopic.php?t=51483
+    https://www.youtube.com/watch?v=AOvwWOGH7bQ&list=PLk-SdLbgd5ol62NX0q7ys5ZbmJFVAMzCF&index=2
+
+
+SOUNDS
+---------------
+
+wind (hackerb9) - CC 0
+    edited by dalas; I release the edited sound under CC BY-SA 4.0
+    https://freesound.org/people/hackerb9/sounds/555246/
+
+gate-fall
+lich_die_wail
+magic-twilight
+magic-twilight-miss
+    made by dalas by modifying pre-existing wesnoth sounds (GPL v2.0 or later)
+
+warhammer (Iwiploppenisse), remixed by dalas (with pre-existing wesnoth sounds) - CC BY-SA 4.0
+    https://freesound.org/people/Iwiploppenisse/sounds/156031/
+
+ASSORTED IMAGES
+===============
+
+ATTACKS
+---------------
+
+blast-wave
+sword-astral-short
+sword-orcish-flaming
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+sword-orcish-swing
+    from the Reign of Lords add-on (GPL v2.0 or later)
+
+
+HALO
+---------------
+
+terror - CC BY-SA 4.0
+    created by dalas
+
+idle-flash
+fire-aura-small
+light-shield
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+shadow-mage-halo - from the Liberty campaign (GPL v2.0 or later)
+
+
+ICONS
+---------------
+
+banana
+drowning
+    made by dalas
+    I release these under CC BY-SA 4.0
+
+coins
+ninja
+ogre
+    made by dalas by modifying pre-existing wesnoth images (GPL v2.0 or later)
+
+planb (Q_x), modified by dalas - GPL 2.0 and GPL 3.0
+    https://opengameart.org/content/dwarven-map
```

Comment:
This one was specifically mentioned in Discord / IRC (both links go to the same discussion):
* https://irclogs.wesnoth.org/2023/07/%23wesnoth-dev.2023-07-28.log
* https://discord.com/channels/231976805987385345/259976436490829825/1134367974887800923

This ought to have a link to the source file in [our resource repo](https://github.com/wesnoth/resources/), and the file added if it's not already there. Althought there's debate about whether the GPL is a good fit when the artist themselves only keeps the .png or .jpeg file, it's clear in this case; the artist has put a .psd file on the OpenGameArt page, and chosen the GPL in 2014, a year after CC BY-SA 4.0 was published.

Reply:
Added PSD and license to the resources directory now.